quarto_demo/quarto/quarto_iod_analysis.qmd

```diff
@@ -0,0 +1,112 @@
+---
+title: "IoD Geographical Analysis"
+title-block-banner: "#0000FF"
+title-block-banner-color: "#FFFFFF"
+toc: true
+toc-location: left
+number-sections: true
+format:
+  html:
+    # html-math-method: katex
+    # self-contained: true
+    # embed-resources: true
+    echo: false
+    mainfont: Sans-Serif
+    grid:
+        body-width: 1000px
+        margin-width: 175px
+        sidebar-width: 150px
+        gutter-width: 1.5em
+include-in-header:
+  - text: |
+     <link rel = "shortcut icon" href = "images/favicon.ico" />
+execute:
+  warning: false
+  echo: false
+jupyter: python3
+---
+```{python}
+# Setting the working directory to be up two levels from the notebooks folder (you shouldn't need to do this in the cookiecutter)
```

Comment:
Any suggestions for a better way of doing this to change the directory. For some reason, outside of cookiecutter it is unable to find the other folders without changing directory.